docs/installation.md

```diff
@@ -50,6 +50,18 @@ To update ty, use `uv tool upgrade`:
 uv tool upgrade ty
 ```
 
+### Installing globally with mise
+
```

Comment:
I think this should come after `pipx`, but definitely after the standalone installer.